externs/browser/w3c_indexeddb.js

```diff
@@ -406,19 +460,24 @@ IDBDatabase.prototype.transaction = function(storeNames, mode) {};
 IDBDatabase.prototype.close = function() {};
 
 /**
- * @type {Function}
+ * @type {?function(!Event)}
```

Comment:
Nevermind. This is OK. It seems to be the way we generally type things the spec says should be `EventHandler`.